mixed/flink/flink-common/src/main/java/com/netease/arctic/flink/catalog/factories/ArcticCatalogFactoryOptions.java

```diff
@@ -31,6 +31,7 @@
 @Internal
 public class ArcticCatalogFactoryOptions {
   public static final String IDENTIFIER = "arctic";
```

Comment:
The `arctic` is deprecated, we shoud use `mixed_iceberg` or `mixed_hive` instead.

Reply:
Thanks for your reminding. I'll open a new PR to fix it.

Reply:
@baiyangtx  I've added a new PR #2424. PTAL.